### PR TITLE
[EDU-3806] Bot update

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-3806]

### Changes

- Adjusting naming.
- Updating how-to guide.

### Additional links

n/a.

[EDU-3806]: https://aziontech.atlassian.net/browse/EDU-3806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx [hunk 1]
```diff
@@ -98,109 +81,170 @@ Where:
...
+:::
+
+### Using Real-Time Events
+
+To access **Real-Time Events**, proceed as follows:
+
+Access [Azion Console](https://console.azion.com/) > **Real-Time Events**.
+In **Data Source**, select **Edge Functions**.
+Define the **Time Filter**, according to the period you want to analyze.
+Use the **Filter by** bar to create a more specific search.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx:225]:
```suggestion
1. Access [Azion Console](https://console.azion.com/) > **Real-Time Events**.
2. In **Data Source**, select **Edge Functions**.
3. Define the **Time Filter**, according to the period you want to analyze.
4. Use the **Filter by** bar to create a more specific search.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 2]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+:::
+
+### Uso do Real-Time Events
+
+Para acessar o **Real-Time Events** proceda da seguinte forma:
+
+[Acesse o Azion Console](https://console.azion.com/) > **Real-Time Events**.
+Em **Data Source**, selecione **Edge Functions**.
+Defina o **Time Filter**, de acordo com o periodo que você deseja analisar.
+Use a barra **Filter by** para crear uma busca mais específica.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:223]:
```suggestion
1. [Acesse o Azion Console](https://console.azion.com/) > **Real-Time Events**.
2. Em **Data Source**, selecione **Edge Functions**.
3. Defina o **Time Filter**, de acordo com o periodo que você deseja analisar.
4. Use a barra **Filter by** para crear uma busca mais específica.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx [hunk 3]
```diff
@@ -98,109 +81,170 @@ Where:
...
 4. Select a *criteria* to run and catch the domains that you want to run the integration on.
     - Use this rule: `if Request URI does not match "\.(png|jpg|css|js|jpeg|gif|ico|ttf|svg|woff|woff2|ashx|asmx|svc|swf|otf|eot)(\?.*)?$"`
-        - This rule is necessary to exclude all static data on your application.
-5. You have to create another criteria for this integration to work: `if Request URI does not match /.well-know/`
-    - This rule is necessary to create a list of allowed IPs that doesn't impact automation or scripts to WEB API.
-6. Below, select a *behavior* to the *criteria*. In this case, it’ll be **Run Function**.
-7. Select the adequate function according to the name you gave it in the instantiate step.
-8. Click the **Save** button.
+        - This rule is strongly recommended to exclude all static data on your application to be processed by the function. You can customize this rule if needed.
+5. You have to create additional **criteria** for this integration to work: `if Request URI does not match /.well-know/`
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx:109]:
```suggestion
5. You have to create additional *criteria* for this integration to work: `if Request URI does not match /.well-know/`
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx [hunk 4]
```diff
@@ -98,109 +81,170 @@ Where:
...
+
+7. `hold_connection`: holds the request, keeping the connection open for *1 minute* before dropping it. To enable this action, you must declare it as follows:
+
+```json
+  "action": "hold_connection"
 ```
 
+This action doesn't require any additional arguments.
+
+—
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx:187]:
```suggestion
---
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx [hunk 5]
```diff
@@ -98,109 +81,170 @@ Where:
...
+
+## Viewing Logs
+
+You can use [Azion Data Streaming](/en/documentation/products/observe/data-streaming/) and [Azion Real-Time Events](/en/documentation/products/observe/real-time-events/) to get the **Bot Manager** logs and monitor the bot activity in your applications.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx:191]:
```suggestion
You can use [Data Streaming](/en/documentation/products/observe/data-streaming/) and [Real-Time Events](/en/documentation/products/observe/real-time-events/) to get the **Bot Manager** logs and monitor the bot activity in your applications.
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx [hunk 6]
```diff
@@ -98,109 +81,170 @@ Where:
...
+
+:::caution[Warning]
+If a product or module is activated, it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
+:::
+
+### Using Data Streaming
+
+To create a new streaming for Bot Manager activity:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx:199]:
```suggestion
To create a new stream for Bot Manager activity:
```

File: src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx [hunk 7]
```diff
@@ -98,109 +81,170 @@ Where:
...
+1. On the upper-left corner, select **Products menu** > **Edge Firewall** in the **Secure** section.
+2. Select the one related to **Bot Manager**.
+3. Open the **Functions Instances** tab to load the integration's configuration.
+4. Select the function instance to edit. 
+5. Edit the variables in the **Arguments** code box.
+
+Tune **Azion Bot Manager** according to the necessities of your business.
 
-This will load a `JSON` file where you'll be able to tune **Azion Bot Protection** according to the necessities of your business.
+<Button href="/en/documentation/products/guides/secure/manage-bots/" text="go to manage bots guide" variant="secondary">
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/marketplace/integrations/bot-protection.mdx:246]: import do component?

File: src/content/docs/en/pages/main-menu/reference/marketplace/bot-protection-reference.mdx [hunk 8]
```diff
@@ -154,43 +152,20 @@ This action allows for introducing delays in responses in cases where bots attem
...
-  "score": 8
-}
-```
-
-This feature provides comprehensive monitoring, post-event analysis, and tracking of potential threats or suspicious activities. You can use your preferred endpoint, including a direct SIEM and Azion **Data Streaming** connectors to receive the information.
-
-With Data Streaming connectors, you're able to feed your stream processing, SIEM, and big data platforms with the event logs from your applications on Azion in real time. This product allows you to collect your logs from and connect them to the endpoint of your data analysis platforms. You can also decide which data you want to use on your analysis by choosing among the available variables.
+### Logs
 
+The requests will generate logs that can be seen in Azion **Real-Time Events** and Azion **Data Streaming**. By analyzing the logs generated by Bot Manager, you can get insights to understand if any changes in the function instance's JSON Args are needed.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/marketplace/bot-protection-reference.mdx:161]:
```suggestion
The requests will generate logs that can be seen in **Real-Time Events** and **Data Streaming**. By analyzing the logs generated by Bot Manager, you can get insights to understand if any changes in the function instance's JSON Args are needed.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 9]
```diff
@@ -34,16 +32,16 @@ Você pode procurar qualquer integração navegando pelos cards ou digitando uma
 
 ### Configure o Edge Firewall
 
-Para instanciar a integração Azion **Bot Protection**, siga estes passos:
+Para instanciar a integração Azion **Bot Manager**, siga estes passos:
 
-1. No canto superior esquerdo do Azion Console, selecione **Products menu** > **Edge Firewall**, dentro da seção **SECURE**.
-2. Clique no botão **Add Rule Set**.
+1. No canto superior esquerdo do Azion Console, selecione **Products menu** > **Edge Firewall**, dentro da seção **Secure**.
+2. Clique no botão **+ Rule Set**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:38]:
```suggestion
2. Clique no botão **+ Rules Engine**.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 10]
```diff
@@ -98,25 +81,16 @@ Onde:
 | Variável | Tipo | Obrigatório | Descrição |
 |---|---|---|---|
 | `threshold` | Número | Sim | A pontuação máxima que a requisição pode atingir antes que a função execute uma ação. Se não tiver valor, a função não agirá |
-| `action` | String | Sim | A ação a ser tomada pela função sempre que a pontuação da requisição for igual ou maior ao limite definido. Valores possíveis: `deny`, `drop` e `log` (para enviar os dados ao endpoint do SIEM, se integrado) |
-| `log_tag` | String | Não | Uma tag para identificar a instância da função que gerou a requisição |
+| `action` | String | Sim | A ação a ser tomada pela função sempre que a pontuação da requisição for igual ou maior ao limite definido. Valores possíveis: `deny`, `redirect`, `custom_html`, `drop`, `log`, `random_delay` e `hold_connection`. Saiba mais sobre a [configuração de actions](#configuracao-de-actions) |
+| `log_tag` | String | Não | Uma tag para identificar  nos logs a instância da função que gerou a requisição. É recomendado usar tags personalizadas e únicas |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:85]:
```suggestion
| `log_tag` | String | Não | Uma tag para identificar nos logs a instância da função que gerou a requisição. É recomendado usar tags personalizadas e únicas |
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 11]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
 Ainda na página **Edge Firewall**:
 
 1. Selecione a aba **Rules Engine**.
-2. Clique no botão **New Rule**.
+2. Clique no botão **+ Rule Engine**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:103]:
```suggestion
2. Clique no botão **+ Rules Engine**.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 12]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
 4. Selecione um *criteria* (critério) para executar a integração:
    - Use esta regra: `if Request URI does not match "\.(png|jpg|css|js|jpeg|gif|ico|ttf|svg|woff|woff2|ashx|asmx|svc|swf|otf|eot)(\?.*)?$"`
-      - Esta regra é necessária para excluir todos os dados estáticos.
+      - Essa regra é recomendada para excluir todos os dados estáticos da sua aplicação a serem processados pela função. Você pode personalizar esta regra, se necessário.
 5. Você precisa criar outro critério para que esta integração funcione: `if Request URI does not match /.well-know/`
-   - Essa regra é necessária para criar uma lista de permissões de IP que não impacte a automação ou scripts da WEB API.
-6. Abaixo, selecione o *behavior* (comportamento) para os *criteria* (critérios). Nesse caso, será **Run Function**.
-7. Selecione a função **Azion Bot Protection** de acordo com o nome que você deu antes.
-8. Clique no botão **Save**.
+   - Essa regra  é recomendada para criar uma lista de permissões de IP que não impacte a automação ou scripts da WEB API.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:110]:
```suggestion
   - Essa regra é recomendada para criar uma lista de permissões de IP que não impacte a automação ou scripts da WEB API.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 13]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+6. Abaixo, selecione o *behavior* (comportamento) **Run Function**.
+7. Logo selecione a função de acordo com o nome que você deu no passo 3.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:112]:
```suggestion
7. Selecione a função de acordo com o nome que você deu no passo 3.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 14]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+8. Clique no botão **Next**.
 
-Pronto. Agora você tem seus domínios protegidos contra ataques de bot usando a integração Azion **Bot Protection**.
+Pronto. Agora você tem seus domínios protegidos contra ataques de bot usando a integração Azion **Bot Manager**.
 
----
+—
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:117]:
```suggestion
---
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 15]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
 
-## Integre com um SIEM
+### Configuração de actions
 
-Se você precisar dos logs do **Bot Protection**, você pode entrar em uma plataforma SIEM para obter estes dados. Para isso, na aba **Args**, atualize a função e insira os dados da conexão com o SIEM.
+Azion **Bot Manager** é capaz de executar **7 ações diferentes** sempre que a pontuação da requisição for maior ou igual ao limite (threshold) definido. Saiba mais sobre cada action a seguir:
 
-### Exemplo de conexão SIEM
+1. allow`: permite a continuação da requisição. Para habilitar esta ação, você deve declará-la da seguinte forma:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:123]:
```suggestion
1. `allow`: permite a continuação da requisição. Para habilitar esta ação, você deve declará-la da seguinte forma:
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 16]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+
+6. `random_delay`: faz com que a função espere por um período aleatório entre `1` e `10` segundos antes de permitir que a requisição prossiga. Para habilitar esta ação, você deve declará-la da seguinte forma:
+
+```json
+  "action": "random_delay"
+```
+
+Essa ação não requer argumentos adicionais.
+
+7. `hold_connection`: "reném a requisição, mantendo a conexão aberta por *1 minuto* antes de encerrá-la. Para habilitar esta ação, você deve declará-la da seguinte forma:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:179]: retém?

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 17]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+```json
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:180]:
```suggestion

```json
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 18]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+  "action": "hold_connection"
 ```
+Essa ação não requer argumentos adicionais.
+
+—
+
+## Consulte os logs
+
+Você pode usar [Azion Data Streaming](/pt-br/documentacao/produtos/observe/data-streaming/) e [Azion Real-Time Events](/pt-br/documentacao/produtos/observe/real-time-events/) para obter os logs do **Bot Manager** e monitorar a atividade dos bots em suas aplicações.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:189]:
```suggestion
Você pode usar [Data Streaming](/pt-br/documentacao/produtos/observe/data-streaming/) e [Real-Time Events](/pt-br/documentacao/produtos/observe/real-time-events/) para obter os logs do **Bot Manager** e monitorar a atividade dos bots em suas aplicações.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 19]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+5. Em **Template**, selecione **Edge Functions Event Collector** e verá também na caixa de código **Data Set** as variáveis que vão ser usadas na análise de seus logs.
+6. Em **Domains**, selecione entre **Filter Domains** ou **All Current and Future Domains**.
+7. Em **Destination**, selecione um **Endpoint Type** na lista suspensa.
+- Os campos para preenchimento são diferentes dependendo do tipo de endpoint que você escolher. Descubra mais informações sobre cada campo na página de [configuração de endpoint](/pt-br/documentacao/produtos/data-streaming/#endpoints).
+8. Clique no botão **Next**.
+
+Agora você pode consultar os logs no endpoint selecionado. 
+
+:::tip
+Consulte a documentação para dar seus [Primeiros passos do Data Streaming](/pt-br/documentacao/produtos/observe/data-streaming/primeiros-passos/) e [Configurar as main settings do Data Streaming](/pt-br/documentacao/produtos/guias/como-usar-data-streaming/)
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:213]:
```suggestion
Consulte a documentação para dar seus [Primeiros passos do Data Streaming](/pt-br/documentacao/produtos/observe/data-streaming/primeiros-passos/) e [Configurar as main settings do Data Streaming](/pt-br/documentacao/produtos/guias/como-usar-data-streaming/).
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 20]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+:::
+
+### Uso do Real-Time Events
+
+Para acessar o **Real-Time Events** proceda da seguinte forma:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:218]:
```suggestion
Para acessar o **Real-Time Events**, proceda da seguinte forma:
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 21]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+
+1. [Acesse o Azion Console](https://console.azion.com/) > **Real-Time Events**.
+2. Em **Data Source**, selecione **Edge Functions**.
+3. Defina o **Time Filter**, de acordo com o periodo que você deseja analisar.
+4. Use a barra **Filter by** para crear uma busca mais específica.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:223]:
```suggestion
4. Use a barra **Filter by** para criar uma busca mais específica.
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 22]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
+
+:::tip
+Consulte a documentação para dar seus [Primeiros passos do Real⁠-⁠Time Events](/pt-br/documentacao/produtos/observe/real-time-events/primeiros-passos/) e [entender os logs do Real⁠-⁠Time Events](/pt-br/documentacao/produtos/guias/observe/entender-logs/)
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:226]:
```suggestion
Consulte a documentação para dar seus [Primeiros passos do Real⁠-⁠Time Events](/pt-br/documentacao/produtos/observe/real-time-events/primeiros-passos/) e [entender os logs do Real⁠-⁠Time Events](/pt-br/documentacao/produtos/guias/observe/entender-logs/).
```

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx [hunk 23]
```diff
@@ -126,81 +100,150 @@ Para concluir, você precisa configurar o **Rules Engine** e nele configurar o *
...
 A Azion fornecerá configurações básicas e fáceis de usar, que devem ser suficientes para a maioria dos casos. Se você precisar de uma configuração mais detalhada, como regras personalizadas, você pode editar o arquivo `JSON` da integração.
 
 Para encontrar esse arquivo:
 
-1. No canto superior esquerdo do Azion Console, selecione **Products menu** > **Edge Firewall**, dentro da seção **SECURE**.
-2. Selecione o relacionado com **Bot Protection**. 
-3. Abra a aba **Functions** para carregar o formulário contendo o código-fonte da integração.
-4. Selecione a aba **Args**.
+1. No canto superior esquerdo do Console, selecione **Products menu** > **Edge Firewall**, dentro da seção **Secure**.
+2. Selecione o relacionado com o **Bot Manager**.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:238]:
```suggestion
2. Selecione a instância relacionada com o **Bot Manager**. 
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/marketplace/bot-protection-referencia.mdx [hunk 24]
```diff
@@ -155,42 +153,20 @@ Esta ação permite causar atraso nas respostas nos casos em que os bots tentam
...
-  "remote_addr": "34.95.175.91",
-  "score": 8
-}
-```
+### Logs
 
-Esta funcionalidade fornece monitoramento abrangente e análise pós-evento e rastreamento de ameaças potenciais ou atividades suspeitas. Você pode usar seu endpoint preferido, incluindo conectores do **Data Streaming** da Azion e SIEMs, para receber as informações.
-
-Com os conectores do Data Streaming, você pode alimentar seus sistemas de processamento de fluxo, SIEM e plataformas de big data com os logs de eventos de suas aplicações na Azion em tempo real. Este produto permite que você colete seus logs e os conecte ao endpoint de suas plataformas de análise de dados. Você também pode decidir quais dados deseja usar em sua análise, escolhendo entre as variáveis disponíveis.
+As requisições gerarão logs que poderão ser visualizados no Azion **Real-Time Events** e no Azion **Data Streaming**. Ao analisar os logs gerados pelo Bot Manager, você pode obter insights para entender se são necessárias alterações nos argumentos JSON da instância da função.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/marketplace/bot-protection-referencia.mdx:162]:
```suggestion
As requisições gerarão logs que poderão ser visualizados no **Real-Time Events** e no **Data Streaming**. Ao analisar os logs gerados pelo Bot Manager, você pode obter insights para entender se são necessárias alterações nos argumentos JSON da instância da função.
```

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx (hunk 16 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/integrations/bot-protection.mdx:179]:
```suggestion
7. `hold_connection`: retém a requisição, mantendo a conexão aberta por *1 minuto* antes de encerrá-la. Para habilitar esta ação, você deve declará-la da seguinte forma:
```

**Comment by lukasmcardoso91** [comment]
De acordo com os ajustes de nome e outros.

**Pull request merged**